scala_config.bzl

```diff
@@ -39,8 +50,19 @@ def _store_config(repository_ctx):
         "ENABLE_COMPILER_DEPENDENCY_TRACKING=" + enable_compiler_dependency_tracking,
     ])
 
+    build_file_content = "\n".join([
```

Comment:
Same as above. I think multiline string would be cleaner (at least this is what is used in other places)